### ISSUE TITLE
Should the iamges be normalized?

### ISSUE THREAD

**Issue by YushengZhang** [cameronfabbri/Compute-Features#3]
I'm sorry to bother you,but I wanna know whether the images should be normalized before I feed them to the pretrained models?

**Comment by cameronfabbri** [comment]
No, to the best of my knowledge they shouldn't be. It's my understanding the models do the proper preprocessing necessary for each network.

If you are willing to move to Tensorflow 2.0, they offer pretty much this entire repo built in, it's pretty nice. See this [tutorial](https://github.com/tensorflow/models/blob/master/research/nst_blogpost/4_Neural_Style_Transfer_with_Eager_Execution.ipynb) on style transfer for how to use VGG net, as well as the [keras applications](https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/applications) for all of the other ones. Those do have preprocessing functions before extracting features.

**Comment by YushengZhang** [comment]
> No, to the best of my knowledge they shouldn't be. It's my understanding the models do the proper preprocessing necessary for each network.
> 
> If you are willing to move to Tensorflow 2.0, they offer pretty much this entire repo built in, it's pretty nice. See this [tutorial](https://github.com/tensorflow/models/blob/master/research/nst_blogpost/4_Neural_Style_Transfer_with_Eager_Execution.ipynb) on style transfer for how to use VGG net, as well as the [keras applications](https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/applications) for all of the other ones. Those do have preprocessing functions before extracting features.

Thanks for your reply. I'll consider it.
